cmake/modules/ddr/GenerateStub.cmake

```diff
@@ -27,8 +27,26 @@ if(NOT EXISTS "${input_file}")
 	message(FATAL_ERROR "Input file '${input_file}' does not exist")
 endif()
 
-execute_process(COMMAND grep -E -q "@ddr_(namespace|options):" ${input_file} RESULT_VARIABLE rc)
+macro(convert_path output filename)
+	if(PATH_TOOL)
+		execute_process(
+			COMMAND ${PATH_TOOL} "${filename}"
+			OUTPUT_VARIABLE _converted_path
+			RESULT_VARIABLE _convert_rc
+		)
+		if(_convert_rc)
```

Comment:
For consistency, this should say `if(NOT _convert_rc EQUAL 0)` (see lines 50 & 59).